vllm/v1/sample/sampler.py

```diff
@@ -44,8 +43,11 @@ def forward(
         logits = self.apply_allowed_token_ids(logits, sampling_metadata)
         # Apply bad words exclusion.
         logits = self.apply_bad_words(logits, sampling_metadata)
-        # Apply logits bias.
-        logits = self.apply_logits_bias(logits, sampling_metadata)
+
+        # Apply greedy-sampling-compatible logits processors.
```

Comment:
I think  "compatible" is the wrong word here .. these are logits procs which can influence greedy sampling.